sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/AzMonNewListExtensionsTests.cs

```diff
@@ -11,13 +12,11 @@ namespace Azure.Monitor.OpenTelemetry.Exporter.Tests
     public class AzMonNewListExtensionsTests
     {
         [Theory]
-        [InlineData("http", "example.com", "8080", "/search", "q=OpenTelemetry", "http://example.com:8080/search?q=OpenTelemetry")]
-        [InlineData(null, "example.com", "8080", "/search", "q=OpenTelemetry", "example.com:8080/search?q=OpenTelemetry")]
-        [InlineData("http", null, "8080", "/search", "q=OpenTelemetry", null)]
-        [InlineData("http", "example.com", null, "/search", "q=OpenTelemetry", "http://example.com/search?q=OpenTelemetry")]
-        [InlineData("http", "example.com", "8080", null, "q=OpenTelemetry", "http://example.com:8080/?q=OpenTelemetry")]
+        [InlineData("http", "example.com", "8080", "/search", "?q=OpenTelemetry", "http://example.com:8080/search?q=OpenTelemetry")]
```

Comment:
Removed few inline data here as the `server.address`, `url.scheme` is mandate here.